### PR TITLE
Added sachdevlaksh.json in public data

### PR DESCRIPTION
**Pull request by sachdevlaksh**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1744"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍 
Thanks for contributing! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/sachdevlaksh.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      "icon": "graduation-hat",
+      "color": "#699cf7",
+      "description": "Moved to Jaipur Rajasthan from Haryana"
+    },
+    {
+      "title": "First line of code!",
+      "date": "March 2011",
+      "icon": "book",
+      "color": "#dadbe7",
+      "description": "At age 20 got book about Ansi C, started coding simple hello world programs. "
```
Comment (schmelto) [inline public/data/sachdevlaksh.json:43]:
```suggestion
      "description": "At age 20 got book about Ansi C, started coding simple hello world programs."
```

Comment (sachdevlaksh) [reply public/data/sachdevlaksh.json]: @schmelto. Thanks for review comment, I will change.

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

**Pull request merged**